apps/example/android/app/src/main/java/com/example/MainActivity.java

```diff
@@ -36,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
 
     // MoEngage SDK initialisation: Replace "xxxxxxx" with your APP ID.
-    MoEngage moEngage = new MoEngage.Builder(MainActivity.this.getApplication(), "9CPVSGMJ6O0VL57NHN91QMSS")
+    MoEngage moEngage = new MoEngage.Builder(MainActivity.this.getApplication(), "xxxxxxx")
```

Comment:
can the value here be passed down via env variable or env file?

Reply:
I need to explore https://github.com/luggit/react-native-config this basically. Shall I proceed?

Reply:
we already have support for .env files. What is the issue with existing one?

Reply:
Added support to add the MoEngage key from `.env` file.
We don't need to make any changes as the package which I suggested is already implemented in our RN project. So we just need to refer the key e.g., `BuildConfig.MOENGAGE_ANDROID_APP_ID`.

Reply:
fixed in this commit: https://github.com/rudderlabs/rudder-sdk-react-native/pull/155/commits/6361076dbbdccd590e58bbc3e6ee5b75006d842d